src/main/java/teammates/ui/controller/InstructorCourseEditPageData.java

```diff
@@ -7,24 +8,130 @@
 import teammates.common.datatransfer.InstructorAttributes;
 import teammates.common.util.Const;
 import teammates.common.util.Url;
+import teammates.ui.template.CourseEditInstructorPanel;
+import teammates.ui.template.ElementTag;
 
 
 public class InstructorCourseEditPageData extends PageData {
-
-    public InstructorCourseEditPageData(AccountAttributes account) {
+    private int instructorToShowIndex;
+    private ElementTag deleteCourseButton;
+    private CourseAttributes course;
+    private List<CourseEditInstructorPanel> instructorPanelList;
+    private CourseEditInstructorPanel addInstructorPanel;
+    private InstructorAttributes currentInstructor;
```

Comment:
If you're only using `currentInstructor` in the method scopes, maybe pass it around the methods instead of storing it in the class scope?